examples/lib-complex/test/DonationsV1.test.js

```diff
@@ -4,17 +4,12 @@ require('./setup')
 import { Contracts, encodeCall } from 'zos-lib'
```

Comment:
@ajsantander @spalladino @facuspagnuolo I think the import of `encodeCall` here can be removed in light of the changes in this PR

Reply:
Thanks for spotting this, Paul! We'll probably make a cleanup pass over all examples soon, and will take care of this then.